ambry-api/src/main/java/com.github.ambry/rest/RestRequest.java

```diff
@@ -89,4 +92,33 @@ public void close()
    * @return the {@link RestRequestMetricsTracker} instance attached to this RestRequest.
    */
   public RestRequestMetricsTracker getMetricsTracker();
+
+  /**
```

Comment:
moved from `ReadableStreamChannel`. All instances of the word "channel" replaced with "data".